npm/private/npm_package_store.bzl

```diff
@@ -223,16 +223,21 @@ def _npm_package_store_impl(ctx):
                 # tar to strip one directory level. Some packages have directory permissions missing
                 # executable which make the directories not listable (pngjs@5.0.0 for example).
                 bsdtar = ctx.toolchains["@aspect_bazel_lib//lib:tar_toolchain_type"]
-                args = ctx.actions.args()
-                args.add(bsdtar.tarinfo.binary)
-                args.add(src)
-                args.add(package_store_directory.path)  # Need to use `.path` due to: Error in add: Cannot add directories to Args#add since they may expand to multiple values. Either use Args#add_all (if you want expansion) or args.add(directory.path).
-                ctx.actions.run_shell(
-                    tools = [bsdtar.tarinfo.binary],
+                ctx.actions.run(
```

Comment:
Right. Now that we've backed out of the chmod we can use `actions.run` again. Can you mention that this is restoring the previous pattern in the PR summary.